src/test/java/seedu/anilist/model/stats/StatsTest.java

```diff
@@ -112,4 +118,28 @@ public void getStats_withAllGenresInGenreList_returnsTrue() {
         assertEquals(stats.getNumUniqueGenres(), expectedNumUniqueGenres);
         assertEquals(stats.getTopGenres(), expectedTopGenres);
     }
+
+    @Test
+    public void equals() {
+        Genre firstGenre = new Genre(GenreList.getListOfGenres()[0]);
+        Genre secondGenre = new Genre(GenreList.getListOfGenres()[1]);
+        HashMap<Genre, Integer> genres1 = new HashMap<>() {{
+                put(firstGenre, 2);
+                put(secondGenre, 3);
+            }};
+        HashMap<Genre, Integer> genres2 = new HashMap<>() {{
+                put(secondGenre, 3);
+                put(firstGenre, 2);
+            }};
+        Stats stats1 = new Stats(1, 2, 3, genres1);
+        Stats stats2 = new Stats(1, 2, 3, genres2);
+        Stats stats3 = new Stats(0, 0, 0, new HashMap<>());
+
+        //same Stats object
+        assertTrue(stats1.equals(stats1));
+        //different Stats object
+        assertTrue(stats1.equals(stats2));
```

Comment:
can the description be better?

Reply:
Are you referring to something like //differe

Reply:
yeap. so like `//different stats object but identical genre count and episode count`